assets/scripts/index.js

```diff
@@ -10,25 +10,46 @@ const eventEndDate = '2016-06-22 20:00:00 GMT-0400';
 setInterval(function() {
 
     // Animate the Clock Hands
-    const r = (el, deg) => {
-        el.setAttribute('transform', `rotate(${deg} 50 50)`)
+    if(!moment().isAfter(eventEndDate)) {
+        const r = (el, deg) => {
+            el.setAttribute('transform', `rotate(${deg} 50 50)`);
+        }
+        const d = new Date();
+        r(hour, 30 * (d.getHours() % 12) + d.getMinutes() / 2);
+        r(min, 6 * d.getMinutes());
+        r(sec, 6 * d.getSeconds());
+    }
+
+    // Count down to event start date
+    if(moment().isBefore(eventStartDate)) {
+        const days = moment().countdown(eventStartDate).days;
```

Comment:
@singram26 perhaps you should create a method to update the text, where you could pass in startDate or EndDate or 88:88, instead of having these 10 lines repeated 3 times